core/node/eth_sender/src/eth_tx_manager.rs

```diff
@@ -377,54 +398,6 @@ impl EthTxManager {
         Ok(None)
     }
 
-    async fn send_unsent_txs(
```

Comment:
This method has been planned for removal for some time (get_unsent_txs doesn't return new transactions anymore because of setting sent_at_block unconditionally)